docs/image_io_implementations.rst

```diff
@@ -0,0 +1,17 @@
+Implementations
+---------------
+
+Image Reader implementations that use PIL and GDAL.
+
+PilImageReader
+++++++++++++++
+
+.. autoclass:: smqtk_image_io.impls.image_reader.pil_io.PilImageReader
+
```

Comment:
This extra new-line is causing issues sphinx parsing for rendering.

Reply:
really? without it I'm getting this error `SMQTK-Image-IO/smqtk_image_io/impls/image_reader/pil_io.py:docstring of smqtk_image_io.impls.image_reader.pil_io.PilImageReader.valid_content_types:2: WARNING: Field list ends without a blank line; unexpected unindent.`

Reply:
I see that too, but this additional new-line is also an issue. It doesn't show a warning, it just gets parsed in a way that we don't want it to be.

Reply:
I think that warning may stem from the parent class doc-string, so we can ignore that here I think.